<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_aufgabe1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Aufgabe 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_aufgabe2a">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Aufgabe 2 a</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_aufgabe2b">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Aufgabe 2 b</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_aufgabe2c">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Aufgabe 2 c</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_aufgabe2d">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Aufgabe 2 d</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_aufgabe3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Aufgabe 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_aufgabe4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Aufgabe 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_aufgabe5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Aufgabe 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_aufgabe6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Aufgabe 6</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
